examples/chart/teleport-cluster/values.yaml

```
@@ -342,13 +342,19 @@ annotations:
 # Kubernetes service account to create/use.
 serviceAccount:
   # Specifies whether a ServiceAccount should be created
-  create: true 
+  create: true
   # The name of the ServiceAccount to use.
   # If not set and serviceAccount.create is true, the name is generated using the release name.
   # If create is false, the name will be used to reference an existing service account.
   name: ""
   # To set annotations on the service account, use the annotations.serviceAccount value.
 
+# Kubernetes ClusterRole and ClusterRoleBinding to create.
+clusterRole:
+  # Specifies wheather a ClusterRole and ClusterRoleBindigs should be created.
+  # Set to false if your cluster level resources managed separately due to segregation of duties.
```

```suggestion
  # Set to false if your cluster level resources are managed separately due to segregation of duties.
```

---

```suggestion
  # Set to false if your cluster level resources are managed separately.
```

IMO we don't care about the why, we care about the what.